### PR TITLE
Support or-conjunction for root queries

### LINKED ISSUE #1507: Conjunction (`and`) with `none` query seems to not work
On [this page](https://wpt.fyi/results/?diff&filter=ADC&q=none%28status%3Amissing%29%20and%20is%3Adifferent&run_id=320340002&run_id=322080003), the query is `none(status:missing) and is:different` but the `and` doesn't seem to work, all results are returned.

### PR DESCRIPTION
**Pull request by lukebjerring**
## Description
Follow-up for https://github.com/web-platform-tests/wpt.fyi/pull/1500 to fix #1507 

## Review Information
- Visit the deployed environment
- Search for `all(status:pass) or none(status:pass)`
- See expected results

### CODE REVIEW

File: webapp/components/test/test-search.html [hunk 1]
```diff
@@ -361,6 +361,21 @@
           }],
       });
     });
+
+    test('multi-root', () => {
+      assertQueryParse('none(status:missing) count>0(status:!pass)', {
```
Comment (foolip) [inline webapp/components/test/test-search.html]: Can we also support explicit and at this level? Follow-up is fine.

Comment (lukebjerring) [reply webapp/components/test/test-search.html]: Already had, just wasn't documented.

**Comment by lukebjerring** [comment]
searchcache in staging is having issues, but I can see the produced `idlharness all(status:fail)` query is `{"run_ids":[259450014,245520018,235540020,251530023],"query":{"and":[{"exists":[{"pattern":"idlharness"}]},{"all":[{"status":"FAIL"}]}]}}` as it should be, so I'll merge anyway.

**Comment by foolip** [comment]
https://github.com/web-platform-tests/wpt.fyi/pull/1511 was merged, so is this fixed then?

**Pull request merged**